x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/job/results/AutodetectResult.java

```diff
@@ -168,7 +168,7 @@ public void writeTo(StreamOutput out) throws IOException {
         writeNullable(modelSnapshot, out);
         writeNullable(modelSizeStats, out);
         writeNullable(modelPlot, out);
-        if (out.getVersion().onOrAfter(Version.V_8_0_0)) {
+        if (out.getVersion().onOrAfter(Version.V_7_9_0)) {
```

Comment:
As above, it will save work later on just to delete this line now.

Reply:
Done.